libnmstate/netapplier.py

```diff
@@ -63,9 +63,18 @@ def apply(desired_state, verify_change=True, commit=True, rollback_timeout=60):
     validator.validate_dns(desired_state)
     validator.validate_vxlan(desired_state)
 
+    vlans_with_mtu_changes = _prepare_vlan_ifaces_for_mtu_change(desired_state)
```

Comment:
Why do we need the complexity of filtering the vlans with changed mtu?
Is there a problem to be consistent and fetch all vlans, no matter if the mtu changed or not?

Reply:
Yes that is what we are doing actually. This variable name is confusing I will change the name to `vlans_ifaces` or something like that. Thanks for point it out!

Reply:
Done!